### ISSUE TITLE
Multiplayer

### ISSUE THREAD

**Issue by bl3u456** [abluescarab/tModLoader-CheatHotkeys#2]
I am trying to play with my friends, but i dont know how. It wont let me and says lost connection, is there a way to play?

**Comment by bl3u456** [comment]
Do I have to like, do something to enable online playing again?

**Comment by abluescarab** [comment]
Are you sure this is a Cheat Hotkeys issue? Does multiplayer work with Cheat Hotkeys disabled?

**Comment by bl3u456** [comment]
no, it doesnt work unless i dont use it at all and reset the files data
